cce/cce.go

```diff
@@ -38,8 +39,23 @@ func GetClusterNames(projectName string) config.Clusters {
 	return clustersArr
 }
 
-func GetKubeConfig(configParams KubeConfigParams) {
-	kubeConfig := getKubeConfig(configParams)
+func GetKubeConfig(configParams KubeConfigParams, skipKubeTLS bool) {
+	kubeConfig, err := getKubeConfig(configParams)
+	if err != nil {
+		common.OutputErrorToConsoleAndExit(err)
```

Comment:
Can we maybe use `log.Fatal` or so instead of `common.OUtputErrorToConsoleAndExit`?

Reply:
I've made a new PR doing that project-wide [here](https://github.com/iits-consulting/otc-auth/pull/63), so I'll mark this as resolved :D